build/docker/proxy/assets/sbin/init.d/00-config.sh

```diff
@@ -4,3 +4,15 @@ set -e
 cp -fv /etc/nginx/nginx.conf.template /etc/nginx/nginx.conf
 
 sed -i "s;API_HOST_TO_REPLACE;${API_HOST//&/\\&};g" /etc/nginx/nginx.conf
+
+allowed_host_list=${API_ADMIN_ALLOWED_HOSTS}
+allowed_hosts=""
+
+IFS=","
+for allowed_host in $allowed_host_list
+do
+    allowed_host_to_add="\"$allowed_host\" 1\;"
+    allowed_hosts+=$allowed_host_to_add
```

Comment:
Can we convert this line to `allowed_hosts+="\"$allowed_host\" 1\;"` and therefore remove the line above?